cluster/core/src/main/resources/reference.conf

```diff
@@ -3,24 +3,25 @@ lagom.cluster {
   # Enables Akka Cluster Bootstrap.
   # https://developer.lightbend.com/docs/akka-management/current/bootstrap/
   #
-  # In dev-mode this setting will be off, otherwise the default is on.
-  # Moreover, Akka Cluster Bootstrap only works if akka.cluster.seed-nodes is not configured.
-  # It's possible to override that by defining akka.cluster.seed-nodes or set this property to off in
-  # the application.conf
+  # This setting is only honored if `akka.cluster.seed-nodes` is not configured. So if you define
+  # `akka.cluster.seed-nodes` Akka Cluster Bootstrap won't run.
+  # In dev-mode this setting will be off.
   bootstrap.enabled = on
 
-  # The cluster node will join itself if akka.cluster.seed-nodes is not configured.
-  # In dev-mode this setting will be on, otherwise the default is off. It's possible
-  # to override that by defining akka.cluster.seed-nodes or set this property to off in
-  # the application.conf
+  # Instruct the process to form an Akka cluster with itself. This setting is not recommended for production.
+  # The cluster node will only join itself if akka.cluster.seed-nodes is not configured.
+  # In dev-mode this setting will be on, otherwise the default is off.
   join-self = ${lagom.defaults.cluster.join-self}
 
   # Exit the JVM forcefully when the ActorSystem has been terminated.
   # This is by default off, but you may want to turn it on in production
   # to restart the process.
   exit-jvm-when-system-terminated = off
 
-
-
 }
 lagom.defaults.cluster.join-self = off
+
+# Enables Health Checks routes. When combined with Akka Cluster Bootstrap,
+# this will add the readiness check `akka.management.cluster.scaladsl.ClusterMembershipCheck`
+# that returns true when node is either Up or WeaklyUp
+akka.management.http.route-providers += "akka.management.HealthCheckRoutes"
```

Comment:
I no longer think this should be enabled by default.

I think this should be added via `g8` and we should document how to add this route but not add the route automatically ourselves.